wrappers/swift/DiagnosticDataViewer.swift

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Wrapper class over `ODWDiagnosticDataViewer` representing Diagnostic Data Viewer Hook.
+public class DiagnosticDataViewer {
+
+    /// MARK: Initialization Methods
+
+    /**
+    Initializes Data Viewer with a specified machine identifier.
+
+    - Parameters:
+        - machineIdentifier: `String` that contains the machine identifier.
+    */
+    public static func initViewerWithMachineIdentifier(machineIdentifier: String) {
```

Comment:
Can we rename this to something like `instantiateViewer(machineIdentifier:)`, please?